project/src/pages/film/film.tsx

```diff
@@ -1,21 +1,26 @@
 // Библиотеки
 import React from 'react';
 import { Helmet } from 'react-helmet-async';
-import { useParams, Link, redirect } from 'react-router-dom';
+import { useParams, Link, redirect, useSearchParams } from 'react-router-dom';
 
 // Типы
 import { MainProps } from '../../types/film';
 
 // Компоненты
+import Tabs from '../../components/tabs/tabs';
 import CardList from '../../components/card-list/card-list';
 
 export default function Film({ films }: MainProps): JSX.Element {
   const { id } = useParams();
   const filmId = Number(id);
-  const film = films.find((value) => (value.id === filmId));
-  if (!film) {
+  const filmIndex = films.findIndex((value) => (value.id === filmId));
+  if (filmIndex === -1) {
     redirect('not-found');
   }
+  const film = films[filmIndex];
+  const searchTab = useSearchParams()[0];
```

Comment:
Если посмотреть сигнатуру useSearchParams хука он возвращает всегда массив из двух объектов (очень похоже на useState), поэтому и писать его лучше так:
```suggestion
  const [searchParams, setSearchParams]= useSearchParams();
```